charmcraft/templates/init/src/charm.py.j2

```diff
@@ -2,7 +2,51 @@
 # Copyright {{ year }} {{ author }}
 # See LICENSE file for licensing details.
 
-"""Charm the service."""
+"""Charm the service.
+
+To gently guide you through the recommended way to write and deploy your new charm,
+here are some essential links and instructions to get you up-to-speed.
+
+To ease the development iterations on your computer we suggest you to setup a
+microk8s environment [1], in summary it will be:
```

Comment:
Suggestion: combine the first two paragraphs in a shorter introduction:

> This is a quick-start guide to help you write and deploy a new charm. To ease the development, we suggest you to setup a microk8s environment [1] by running:

Reply:
Changed accordingly, thanks!